yara-x.yaml

```diff
@@ -29,37 +29,12 @@ pipeline:
 
   - runs: cargo install cargo-c --features=vendored-openssl --root $HOME/.cargo --target-dir $HOME/.cargo
```

Comment:
non-blocking comment: I did a local build after removing `--features=vendored-openssl` and it seems to be working, I understand this is build-time only and this is not ending up in package. It would be great if we avoid any reliance of `vendored-openssl` 💭

Reply:
Yeah, I was curious if we could switch over to using the cargo-c apk instead of building cargo-c in the package itself.

Reply:
I converted over to depending on the cargo-c wolfi package rather than building cargo-c  from scratch, passes tests locally.